### DISCUSSION TITLE
How to reflash to Original Firmware ?

### DISCUSSION THREAD

**Discussion by tranlan2110** [ludwig-v/wireless-carplay-dongle-reverse-engineering#85] [General]
Hi, 
I've flashed to CFW 2021.04.26_BASED, but It has some error when play music (phone call and Siri are working).

I want to flashed back to OFW  U2W_AUTOKIT_Update_2021.03.06.1355

How can I do that ?

**Comment by ludwig-v** [comment]
You must use November rollback image first

**Reply by tranlan2110** [reply]
I've rename file to U2W_AUTOKIT_Update.img / U2W_Update.img but my Carlinkit did'nt flash that file.

Can you write a Tutorial how to reflash to OFW ?

**Reply by tranlan2110** [reply]
> You must use November rollback image first

Ludwiz, please help us to rollback orginal firmware.
